src/app/components/store-products/store-products.component.html

```diff
@@ -0,0 +1,98 @@
+<section class="my-lg-14 my-8">
+    <div class="container"*ngIf="store$ | async as store">
+      <div class="d-flex flex-row justify-content-between" >
+        <div class="d-flex flex-column">
+          <div>
+            <!-- img -->
+            <!-- img -->
+            <img
+              src="../assets/images/stores-logo/stores-logo-1.svg"
```

Comment:
use absolute path, It's my last comment about this so please check your code after adding the template